tests/Feature/SeasonsTest.php

```diff
@@ -0,0 +1,316 @@
+<?php
+
+namespace Tests\Feature;
+
+
+use App\Contributor;
+use App\Podcast;
+use App\Season;
+
+class SeasonsTest extends ApiTestCase
+{
+    /**
+     * @var string
+     */
+    protected $resourceType = 'seasons';
+
+    /**
+     * Test the search route.
+     */
+    public function testSearch()
+    {
+        // ensure there is at least one season in the database
+        $this->model();
+
+        $this->doSearch()
+            ->assertSearchResponse();
+    }
+
+    /**
+     * Test searching for specific ids
+     */
+    public function testSearchById()
+    {
+        $models = factory(Season::class, 2)->create();
+        // This season should not be returned in the results
+        $this->model();
+
+        $this->doSearchById($models)
+            ->assertSearchByIdResponse($models);
+    }
+
+    /**
+     * Test the create resource route.
+     */
+    public function testCreate()
+    {
+        $model = $this->model(false);
+
+        /** @var Podcast $podcast */
+        $podcast = factory(Podcast::class)->create();
```

Comment:
Looks like this variable isn't being used. Should it be removed or were you intending to use it in some way?

Reply:
This was left over when I was trying to figure out how to create a season that was already attached to a podcast. But I'm pretty sure that the jsonapi spec doesn't allow for it. So it will need to be multiple requests: first creating the season, then adding the season to the podcast.